code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -59,7 +59,7 @@
 
 /obj/item/weapon/grenade/iedcasing/prime() //Blowing that can up
 	update_mob()
-	explosion(src.loc,-1,-1,-1, flame_range = range)	// no explosive damage, only a large fireball.

```

Comment:
Again i'd like to reiterate that i'm really fucking bad at this. I saw the range variable at the start of the file with the rest of the item's stats but i'm not sure changing it would have the desired effect because later on theres this bit of code: 

	if(det_time < 0) //checking for 'duds'
		range = 1
		det_time = rand(30,80)
	else
		range = pick(2,2,2,3,3,3,4)

meaning the first range variable is either redundant and ignored or plays some unknown role, so I decided to just ignore it and force a certain fireball size than toy with it and possibly break it. This appears to have the unintended side effect of removing duds...

Also, sorry for ignoring you, but i'm having trouble keeping up with replies and some suggestions are also beyond my abilities as well.